test/System.IdentityModel.Tokens.Jwt.Tests/CreateAndValidateTokens.cs

```diff
@@ -345,7 +345,7 @@ public static TheoryData<JwtTheoryData> RoundTripTokensUsingCacheTheoryData()
             var theoryData = new TheoryData<JwtTheoryData>();
             var handler = new JwtSecurityTokenHandler();
             var asymmetricSecurityTokenDescriptor = Default.AsymmetricSignSecurityTokenDescriptor(null);
-            var cryptorProviderFactory = new CryptoProviderFactory { CacheSignatureProviders = true };
+            var cryptorProviderFactory = new CryptoProviderFactory();
```

Comment:
Can we add a couple of tests that turn caching off and make sure new signature providers are created and released?

Reply:
Good idea. I added a few test cases for this, but let me know if we need more comprehensive tests.